api/providers/views.py

```diff
@@ -595,3 +596,27 @@ def perform_destroy(self, instance):
             self.get_provider().remove_from_group(instance, instance.permission_group)
         except ValueError as e:
             raise ValidationError(str(e))
+
+
+class RegistrationProviderSchemaList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticated,
+        base_permissions.TokenHasScope,
+        MustBeModerator,
```

Comment:
This doesn't seem right. Shouldn't everyone have access to the list of schemas available to a registration provider? The use case is that, when you're creating a registration for a registration provider, the front end gets a list of the schemas available for that provider, and then shows that list as possible options to the user.

Reply:
Ok, this will be totally public, but only for visible and up to date schemas? or all associated schemas?

Reply:
Hmm. I _think_ visible and up-to-date. Basically whatever you'd expect to see when you're making a registration and it shows you which schemas you have to choose from.